docs/platform/concepts/byoc.md

```diff
@@ -110,37 +110,45 @@ traffic through a proxy for additional security. To accomplish this, Aiven
 utilizes a bastion host (**Bastion node**) physically separated from the Aiven services
 you deploy. The service VMs reside in a privately addressed subnet (**Private subnet**)
 and are accessed by the Aiven management plane via the bastion. They are not
-accessible through the Internet.
+accessible through the internet.
 
 :::note
 Although the bastion host and the service nodes reside in the VPC under
 your management (**BYOC VPC**), they are not accessible (for example, via SSH) to anyone
 outside Aiven.
 
-The bastion and workload nodes require outbound access to the Internet
+The bastion and workload nodes require outbound access to the internet
 to work properly (supporting HA signaling to the Aiven management node and RPM download
 from Aiven repositories).
 :::
 
+The private subnet is where your object storage (two S3 buckets) reside. This storage is
```

Comment:
Actually the S3 buckets are accessible through AWS public API. Which means they are technically accessed through the internet (or at least through AWS global backbone network). However, we have a ticket in MS3 to use VPC S3 interface endpoint to be able access the S3 bucket using a private IP address.

Reply:
The buckets aren't in the subnets indeed, however @allaouiamine we already use an S3 endpoint and that's how these buckets are accessed, especially as they are in the same region.

Reply:
@allaouiamine @mlatief so if outside the VPC, where do the S3 buckets reside and do we need any connector lines to show how they connect to whatever? At the moment, they are located in the VPC and don't connect to any other network element. How do I fix that?

<img width="885" alt="Screenshot 2024-10-18 at 13 30 02" src="https://github.com/user-attachments/assets/1049832b-187c-40d8-86ff-05424011b680">

Reply:
Hi @wojcik-dorota,
1. The S3 buckets reside in the customer's account but outside the VPC.
2. The S3 buckets are in the customer's account. But then they are accessed through the public internet.
I suggest you put the S3 buckets below (or at the end of) "Your AWS account" in the diagram and outside "BYOC VPC". and then use another line from "Public Internet" to the S3 buckets.

Reply:
@allaouiamine @mlatief Fixed! Please have another look :-)